### DISCUSSION TITLE
Preventivo, note documento a piè pagina

### DISCUSSION THREAD

**Discussion by mastsolutions** [fattureincloud/api#202] [Q&A] [answered]
Buongiorno,
con API v1 passavo le note documento a piè pagina, non riesco a capire come fare la stessa cosa con API v2 (C#)
potete aiutarmi?
Grazie Stefano

**Comment by emajo** [comment, marked as answer]
Ciao @mastsolutions il campo a cui ti riferisci dovrebbe essere `notes`, per essere sicuri ti consiglio di fare una richiesta GET con API v2 di un documento creato con API v1 che presenta le note a cui ti riferisci, per andare a vedere il campo che effettivamente contiene le note a cui ti riferisci.

Facci sapere se hai risolto,
grazie,
Emanuele.
